batch/2.2/_index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Jakarta Batch 2.2 (Under Development)"
+date: 2025-04-15
+summary: "Enhancements for Jakarta EE 12"
+---
+
+<!-- Please describe the high-level changes made to Jakarta Wombat 1.0. --> 
+<!-- The intent is for the first two sections to be an executive summary in the range of 300 to 800 characters. -->
+<!-- Links can accompany the executive summary, but cannot substitute for an executive summary. -->
+
+This release aims to remove references to SecurityManager.
+
+### New features, enhancements or additions
+<!-- List here -->
+* Remove SecurityManager references from API
+
+### Removals, deprecations or backwards incompatible changes
+<!-- List here -->
+* None
```

Comment:
As explained [here](https://github.com/jakartaee/batch/issues/215) it is only the API JAR implementation, not the spec API nor spec document, which needs to be changed.